src/js/edu-benefits/5495/config/form.js

```diff
@@ -0,0 +1,56 @@
+// import _ from 'lodash/fp';
+
+// import fullSchema5495 from 'vets-json-schema/dist/dependents-change-of-program-schema.json';
```

Comment:
i have a pr open for the name change https://github.com/department-of-veterans-affairs/vets-json-schema/pull/78